src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -227,9 +235,27 @@ public override StandardValuesCollection GetStandardValues(ITypeDescriptorContex
                 // We need to get the enum values in this rather round-about way so we can filter
                 // out fields marked Browsable(false). Note that if multiple fields have the same value,
                 // the behavior is undefined, since what we return are just enum values, not names.
-                Type reflectType = TypeDescriptor.GetReflectionType(EnumType) ?? EnumType;
+                // Given that EnumType is constrained to be an enum, we suppress calls for reflection with Enum.
+
+                [UnconditionalSuppressMessage("Trimming", "IL2067:", Justification = "Trimmer does not trim Enums")]
+                [return: DynamicallyAccessedMembers(TypeDescriptor.ReflectTypesDynamicallyAccessedMembers)]
```

Comment:
Do you know why [TypeDescriptor.ReflectTypesDynamicallyAccessedMembers](https://github.com/dotnet/runtime/blob/e629779764abd70bde54e780eeeb8fd5bee84fac/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs#L22) is defined as
```DynamicallyAccessedMemberTypes.PublicParameterlessConstructor | DynamicallyAccessedMemberTypes.PublicFields;```
instead of
```DynamicallyAccessedMemberTypes.PublicParameterlessConstructor | DynamicallyAccessedMemberTypes.PublicProperties;``` ?

cc @eerhardt

Reply:
I believe it is because these were the only things that were needed to be preserved because this was only called in 2 places:

https://github.com/dotnet/runtime/blob/470df6172572054bdcdd4f798f9a34abd282dd49/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs#L260-L269

https://github.com/dotnet/runtime/blob/470df6172572054bdcdd4f798f9a34abd282dd49/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs#L230-L232

Since these places only needed ctor and fields, that is all that was guaranteed to be preserved.